core/templates/pages/subtopic-viewer-page/navbar-pre-logo-action/subtopic-viewer-navbar-pre-logo-action.component.spec.ts

```diff
@@ -15,38 +15,47 @@
 /**
  * @fileoverview Unit tests for the subtopic viewer pre logo action
  */
-import { TestBed } from '@angular/core/testing';
+
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { SubtopicViewerNavbarPreLogoActionComponent } from
+  './subtopic-viewer-navbar-pre-logo-action.component';
 import { UrlService } from 'services/contextual/url.service';
 
-require(
-  'pages/subtopic-viewer-page/navbar-pre-logo-action/' +
-  'subtopic-viewer-navbar-pre-logo-action.component.ts');
+describe('subtopic viewer pre logo action component', () => {
+  let component: SubtopicViewerNavbarPreLogoActionComponent;
+  let fixture: ComponentFixture<SubtopicViewerNavbarPreLogoActionComponent>;
+  let urlService = null;
```

Comment:
missing type, and you don't have to assign null

Reply:
done